docs/extensibility/internals/initialization-sequence-of-project-subtypes.md

```diff
@@ -25,7 +25,7 @@ The environment constructs a project by calling the base project factory impleme
 
     1.  The environment's implementation of the <xref:Microsoft.VisualStudio.Shell.Interop.IVsCreateAggregateProject.CreateAggregateProject%2A> method calls the `HrCreateInnerProj` method with the following function declaration:
 
-         <CodeContentPlaceHolder>0</CodeContentPlaceHolder>
+         \<CodeContentPlaceHolder>0</CodeContentPlaceHolder>
```

Comment:
These ones I wasn't sure if they were placeholders that didn't get replaced during a migration